cmd/secret/cmd/root_server_postgres_test.go

```diff
@@ -0,0 +1,199 @@
+package cmd
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"net/http/httptest"
+	"os"
+	"strconv"
+	"testing"
+	"time"
+
+	api "github.com/go-itools-internship/go-secret/pkg/http"
+	"github.com/stretchr/testify/require"
+)
+
+func TestRoot_Server_Postgres(t *testing.T) {
+	t.Run("set by key", func(t *testing.T) {
+		t.Run("expect postgres set method success", func(t *testing.T) {
+			ctx, cancel := context.WithTimeout(context.Background(), 15*time.Second)
+			defer cancel()
+			defer func() {
+				err := migrateDown()
+				if err != nil {
+					fmt.Println("Can't migrate down", err)
+				}
+			}()
+			port, err := GetFreePort()
+			require.NoError(t, err)
+			r := New()
+			r.cmd.SetArgs([]string{"server", "--port", strconv.Itoa(port), "--postgres-url", postgresURL, "--migration", "file://../../../"})
```

Comment:
You could reuse `--migration` value, because it has the same value across all tests